### PR TITLE
Add navigation bar

### PR DESCRIPTION
**Pull request by lil2400**
https://trello.com/c/Hz9JCVTx/51-primary-navigation-and-header

### CODE REVIEW

File: src/client/components/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import MenuItem from "@material-ui/core/MenuItem";
+import { makeStyles } from "@material-ui/core/styles";
+import logoSource from "../assets/images/Logo.png";
+
+const useStyles = makeStyles({
+  grow: {
+    flexGrow: 1
+  },
+  toolbarColor: {
+    backgroundColor: "#FFFFFF"
```
Comment (zkwsk) [inline src/client/components/NavBar.js]: Can you make it `#0b3c61`

File: src/client/components/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+const NavBar = () => {
+  const classes = useStyles();
+  return (
+    <div className={classes.grow}>
+      <AppBar position="static">
+        <Toolbar className={classes.toolbarColor}>
+          <div className="Logo">
+            <img src={logoSource} width="400px"></img>
+          </div>
```
Comment (zkwsk) [inline src/client/components/NavBar.js]: Let's remove the logo. I know it was in the wireframe, but the logo we have is not suited for that type of use.

**Review by zkwsk — CHANGES_REQUESTED** [review]

**Review by Swapna-Sahu — DISMISSED** [review]
looks good to me

File: src/client/components/NavBar.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+const NavBar = () => {
+  const classes = useStyles();
+  return (
+    <div className={classes.grow}>
+      <AppBar position="static">
+        <Toolbar className={classes.toolbarColor}>
+          <div className={classes.grow} />
+          <MenuItem className={classes.MenuItemColor}>MAP</MenuItem>
+          <MenuItem className={classes.MenuItemColor}>START VOYAGES</MenuItem>
+          <MenuItem className={classes.MenuItemColor}>ALL VOYAGES</MenuItem>
```
Comment (zkwsk) [inline src/client/components/NavBar.js]: Please make these links active:

```
<MenuItem className={classes.MenuItemColor}><Link to="/voyages">MAP</Link></MenuItem>
<MenuItem className={classes.MenuItemColor}><Link to="/voyages-summary">START VOYAGE</Link></MenuItem>
<MenuItem className={classes.MenuItemColor}><Link to="/voyages-summary">ALL VOYAGES</Link></MenuItem>
```

**Review by zkwsk — CHANGES_REQUESTED** [review]
- [x] Please make the links active

- [x] Please add the component to the top of the App.js component

**Review by zkwsk — APPROVED** [review]

**Pull request merged**